Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -2464,6 +2464,22 @@ final class BuildPlanTests: XCTestCase {
         try testBinaryTargets(platform: "macos", arch: "arm64e", destinationTriple: arm64eTriple)
     }
 
+    func testCreatingSanitizers() throws {
+        for sanitizer in Sanitizer.allCases {
+            XCTAssertEqual(sanitizer, try Sanitizer(argument: sanitizer.shortName))
+        }
+    }
+
+    func testInvalidSanitizer() throws {
+        do {
+            _ = try Sanitizer(argument: "invalid")
+            XCTFail("Should have failed to create Sanitizer")
+        } catch let error as ArgumentConversionError {
+            XCTAssertEqual(
+                error.description, "valid sanitizers: address, thread, undefined, scudo")
+        }
+    }
+
```

Comment:
Another argument for [the suggested change](https://github.com/apple/swift-package-manager/pull/2988#discussion_r508488684) is that test coverage would necessarily follow from the command-line interface. I think an integration test of that sort would be more in line with the rest of this test case than the current API unit test.